zwave_js_server/model/node.py

```diff
@@ -454,12 +454,11 @@ def handle_value_removed(self, event: Event) -> None:
     def handle_value_notification(self, event: Event) -> None:
         """Process a node value notification event."""
         # append metadata if value metadata is available
+        value_notification = ValueNotification(self, event.data["args"])
         value = self.values.get(_get_value_id_from_dict(self, event.data["args"]))
         if value:
             value.update(event.data["args"])
```

Comment:
So the only open question that I am not clear on is whether we still need this line

Reply:
@marcelveldt sorry to keep tagging you, I just didn't fully understand your comment in the other PR. Should we update the `Value` or should we not? I think from a practical standpoint what we'd be updating is the stored value of the `Value`

Reply:
given that this doesn't change functionality by leaving it in, I think we can merge and address this separately if needed